addons/Text_To_Speech/src/presenter.js

```diff
@@ -495,6 +495,19 @@ function AddonText_To_Speech_create() {
         return splitText;
     }
 
+    presenter.removeLonelyPunctuationMarks = function (texts) {
+        const punctuationMarks = ['.', ',', ';', '?', '!', '(', ')', "[", ']', '{', '}', '-', '"', '\''];
```

Comment:
z tego zestawienia powinny zostać usunięte znaki "." oraz "," gdyż są obsługiwane w innej metodzie zapobiegającej czytanie nadmiarowych znaków,
dodatkowo, czy usunięcie ich nie wpłynie na prędkość odczytu tj. czy TTS będzie robić przerwy

Reply:
Ad.1. Tzn, metoda o której wspomniałeś nie naprawia istniejącego problemu. Czy miałeś na myśli, abym się przypatrzył istniejącej już metodzie, aby po dodaniu tej metody się ona nie popsuła ?
Ad.2. Muszę się upewnić, dzięki za zwrócenie na to uwagi.

Reply:
Ad.2. Ciężko było mi znaleźć rzeczywisty przypadek, wiec na sztywno sprawdziłem takowy. TTS robi przerwę, jednak jest ona nieco krótsza w porównaniu do innych przeglądarek np. firefox'a. Jednak jest ona odczuwalna. Kod w tej wersji jaką sprawdzałeś aktywuje się tylko na Chrome, więc nie wpłynie on negatywnie na przerwy.